### ISSUE TITLE
there is no ZCOUNT command

### ISSUE THREAD

**Issue by gihnius** [hoisie/redis#31]
can count a sorted set by zcount key -inf +inf ?

**Comment by alexanderscott** [comment]
Yes, you can do so with `client.ZcountAll` (or provide custom min/max as floats).  This can be seen in implementation [here](https://github.com/hoisie/redis/blob/master/redis.go#L988) and I have opened a pull with appropriate tests for `Zcount` and `ZcountAll` in #43 .
